### PR TITLE
adding parameters for gw levels to data type mapping

### PR DESCRIPTION
**Pull request by ssoper-usgs**
Before making a pull request
----------------------------

- [ ] Make sure all tests run
- [ ] Update the changelog appropriately

Title
-----------
Brief description of changes. Reference the JIRA ticket if appropriate

Description
-----------
If no JIRA ticket is referenced, describe the changes made. Note anything that you want the reviewers to know while
reviewing your pull request

After making a pull request
---------------------------
- [ ] If appropriate, put the link to the PR in the JIRA ticket
- [ ] Assign someone to review unless the change is trivial

### CODE REVIEW

File: liquibase/changeLogs/capture/data/dataTypeMapping/dataTypeMapping.tsv [hunk 1]
```diff
@@ -166,4 +166,21 @@ parm_cd	stat_cd	data_type
 72232	00021	tsDailyValueStatisticalTransform
 72232	00022	tsDailyValueStatisticalTransform
 72232	00023	tsDailyValueStatisticalTransform
-72232	00024	tsDailyValueStatisticalTransform
\ No newline at end of file
+72232	00024	tsDailyValueStatisticalTransform
```
Comment (dsteinich) [inline liquibase/changeLogs/capture/data/dataTypeMapping/dataTypeMapping.tsv:169]: This is an example of (me) not putting the blank line in and the compare showing a false positive change - line 169's real value didn't change, we just added new stuff on new lines after it. One more reason to always have the blank line (unless the load would choke on it).

File: liquibase/changeLogs/capture/data/dataTypeMapping/dataTypeMapping.tsv [hunk 2]
```diff
@@ -166,4 +166,21 @@ parm_cd	stat_cd	data_type
...
+61055	null	discreteGroundWaterTransform
+72150	null	discreteGroundWaterTransform
+62611	null	discreteGroundWaterTransform
+62613	null	discreteGroundWaterTransform
+62610	null	discreteGroundWaterTransform
+62612	null	discreteGroundWaterTransform
+62600	null	discreteGroundWaterTransform
+62601	null	discreteGroundWaterTransform
+30210	null	discreteGroundWaterTransform
+72019	null	discreteGroundWaterTransform
```
Comment (dsteinich) [inline liquibase/changeLogs/capture/data/dataTypeMapping/dataTypeMapping.tsv]: (IOW-453 has 99019 - do we need that here also?) 
ACTUALLY, based on yesterday's conversation everything is under "61055" "Water level, depth MP" "ft", and we should probably only have that.

**Review by dsteinich — CHANGES_REQUESTED** [review]

File: liquibase/changeLogs/capture/data/dataTypeMapping/dataTypeMapping.tsv (hunk 1 above)
Comment (ssoper-usgs) [reply liquibase/changeLogs/capture/data/dataTypeMapping/dataTypeMapping.tsv:169]: I noticed this too, and was annoyed by it (the false positive).  +1 to keeping the blank lines moving forward.

File: liquibase/changeLogs/capture/data/dataTypeMapping/dataTypeMapping.tsv (hunk 2 above)
Comment (ssoper-usgs) [reply liquibase/changeLogs/capture/data/dataTypeMapping/dataTypeMapping.tsv]: I think 99019 doubles up on the value of another parm in the aq_to_nwis_parm table.  I was confused by that so I left it out on the first draft.

File: liquibase/changeLogs/capture/tables/discreteGroundWater/discreteGroundWater.sql [hunk 3]
```diff
@@ -11,6 +11,7 @@ create unlogged table if not exists ${AQTS_SCHEMA_NAME}.discrete_ground_water
 ,completed_work                          text
 ,last_modified                           timestamp
 ,parameter                               text
+,parm_cd                                 text
```
Comment (ssoper-usgs) [inline liquibase/changeLogs/capture/tables/discreteGroundWater/discreteGroundWater.sql:14]: Added this here, but can do a more formal changeset as well.  My thinking is that the capture.discrete_ground_water table is only on test, and not being used by anything yet, we might as well drop and replace.

**Review by dsteinich — APPROVED** [review]

**Pull request merged**